### PR TITLE
Add livestream player

### PR DESCRIPTION
**Pull request by robdodson**
@ebidel @jeffposnick PTAL

### CODE REVIEW

File: summit/app/index.html [hunk 1]
```diff
@@ -65,8 +65,8 @@
 
             <div class="flex horizontal center-center layout">
               <iron-selector class="wide-only" selected="[[page]]" activate-event="click" attr-for-selected="name">
-                <a name="schedule" class="tab" href="/summit/schedule">SCHEDULE</a>
-                <a name="speakers" class="tab" href="/summit/speakers">SPEAKERS</a>
+                <a name="schedule" on-tap="_pauseLivestream" class="tab" href="/summit/schedule">SCHEDULE</a>
```
Comment (jeffposnick) [inline summit/app/index.html]: Are there any other ways besides `tap`-ing on those links that someone might navigate off the page hosting the livestream that would also necessitate calling `_pauseLivestream()`?

Maybe you can listen for a change to `page` and use that as a signal to pause?

File: summit/bower.json [hunk 2]
```diff
@@ -13,20 +13,21 @@
     "tests"
   ],
   "dependencies": {
-    "polymer": "polymer/polymer#^1.0.0",
     "app-layout": "polymerlabs/app-layout#^0.0.2",
     "font-roboto": "polymerelements/font-roboto#^1.0.0",
+    "google-youtube": "GoogleWebComponents/google-youtube#~1.1.2",
```
Comment (jeffposnick) [inline summit/bower.json]: This is using `~` while everything else uses `^`.

File: summit/bower.json [hunk 3]
```diff
@@ -13,20 +13,21 @@
...
     "iron-collapse": "polymerelements/iron-collapse#^1.0.0",
+    "iron-flex-layout": "polymerelements/iron-flex-layout#^1.0.0",
     "iron-icon": "polymerelements/iron-icon#^1.0.0",
     "iron-icons": "polymerelements/iron-icons#^1.0.0",
-    "iron-flex-layout": "polymerelements/iron-flex-layout#^1.0.0",
     "iron-media-query": "polymerelements/iron-media-query#^1.0.0",
     "iron-pages": "polymerelements/iron-pages#^1.0.0",
     "iron-selector": "polymerelements/iron-selector#^1.0.0",
     "neon-animation": "polymerelements/neon-animation#^1.0.0",
+    "page": "visionmedia/page.js#~1.6.3",
```
Comment (jeffposnick) [inline summit/bower.json]: This is using `~` while everything else uses `^`.

**Comment by jeffposnick** [comment]
Actual `<google-youtube>` usage seems :+1:

File: summit/app/index.html (hunk 1 above)
Comment (ebidel) [inline summit/app/index.html]: does tap work well for a11y?

Comment (robdodson) [inline summit/app/index.html]: @ebidel - yeah it seems to work for keyboard and virtual click

**Comment by robdodson** [comment]
Rebased. Per @jeffposnick's suggestion I'm hooking into the router now so whenever the path changes, if the user is leaving the homepage to look at a different section and the livestream is active it'll attempt to pause.

**Comment by ebidel** [comment]
:+1:

**Comment by robdodson** [comment]
Realizing that I accidentally pushed a previous version of these changes with a separate update to the summit site. Going to close this PR and fixup the master branch to match the final state of this PR.

**Pull request closed without merging**